<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Suhanth" failures="3" tests="9" name="Test1" time="51.462" errors="0" timestamp="21 Sep 2024 22:15:21 GMT">
  <testcase classname="com.ajio.women" name="Click_On_Women" time="3.913">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
Assertion Failure">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Assertion Failure
	at com.ajio.women.Click_On_Women(women.java:75)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- Click_On_Women -->
  <testcase classname="com.ajio.women" name="Click_On_Women" time="8.806">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
Assertion Failure">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Assertion Failure
	at com.ajio.women.Click_On_Women(women.java:75)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- Click_On_Women -->
  <testcase classname="com.ajio.women" name="Click_On_Women" time="1.154">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
Assertion Failure">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Assertion Failure
	at com.ajio.women.Click_On_Women(women.java:75)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- Click_On_Women -->
  <testcase classname="com.ajio.women" name="Validate_Women_Page" time="0.453"/>
  <testcase classname="com.ajio.women" name="click_On_BIBA" time="3.692"/>
  <testcase classname="com.ajio.women" name="validate_BIBA_Productpage" time="1.34"/>
  <testcase classname="com.ajio.women" name="nextpage_on_BIBA" time="0.173"/>
  <testcase classname="com.ajio.women" name="nextpage_on_BIBA" time="0.175"/>
  <testcase classname="com.ajio.women" name="nextpage_on_BIBA" time="0.138"/>
</testsuite> <!-- Test1 -->
